pkg/platform/runtime2/runtime.go

```diff
@@ -55,7 +61,16 @@ func (r *Runtime) Update(msgHandler setup.MessageHandler) error {
 }
 
 func (r *Runtime) Environ(inherit bool) (map[string]string, error) {
-	return r.store.Environ(inherit)
+	env, err := r.store.Environ(inherit)
+	if !r.envAccessed {
+		if err != nil {
+			analytics.EventWithLabel(setup.CatRuntime, setup.ActFailure, setup.LblEnv)
```

Comment:
I'd like the dependency on `setup` to be razor thin. Can we either move these constants to the analytics package or just create some shared types package for the runtime stuff?

Reply:
I'll move it to the analytics package.  Most of our event strings live there already...